bin/inlong-daemon

```diff
@@ -32,6 +32,7 @@ command_help() {
     dashboard           Run a Dashboard Service
     dataproxy           Run a Dataproxy Service
     agent               Run a Agent Service
+    sortstandalone      Run a Sortstandalone Service
```

Comment:
Run a Sortstandalone Service
->
Run a Sort Standalone Service

Since `Sortstandalone` is not a readable word.